lms/static/scripts/frontend_apps/components/ErrorDialog.js

```diff
@@ -18,7 +18,7 @@ import ErrorDisplay from './ErrorDisplay';
  */
 export default function ErrorDialog({
   onCancel,
-  description = 'Error',
+  description,
```

Comment:
No longer needed as a somewhat stinky workaround to some of the gymnastics that have been removed from `ErrorDisplay`